### PR TITLE
leadearboard referral data table

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "axios": "^1.2.1",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
+    "console": "^0.7.2",
```
Comment (crazzywizard) [inline package.json]: why do we need this?

**Review by crazzywizard — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (towardsnewtech) [reply package.json:32]: image
In many code, I can see importing part about this lib.

Comment (towardsnewtech) [reply package.json]: removed 🚮

File: components/LeaderboardPage/LeaderboardPage.tsx [hunk 2]
```diff
@@ -1,37 +1,39 @@
...
+import getReferralLeaderboard from "../../lib/getReferralLeaderboard"
+import { getOwnersofCre8ors } from "../../lib/cre8or"
+import getTwitterHandleByAddress from "../../lib/getTwitterHandleByAddress"
 
 const LeaderboardPage = () => {
   const [collectors, setCollectors] = useState([])
-  const isMobile = useMediaQuery("(max-width: 768px)")
 
   useEffect(() => {
     const fetchTopCollectors = async () => {
```
Comment (sweetmantech) [inline components/LeaderboardPage/LeaderboardPage.tsx]: we're not getting collectors anymore right?

might be good to refactor this fx name ✏️ 

```suggestion
    const fetchLeaderboard = async () => {
```

File: components/LeaderboardPage/LeaderboardPage.tsx [hunk 3]
```diff
@@ -86,7 +88,7 @@ const LeaderboardPage = () => {
                     text-[8px] xs:text-[11px] md:text-[18px]
                     w-[100px] xs:!w-[130px] md:!w-[200px]"
                   >
-                    # of NFTs {!isMobile ? "Owned" : ""}
+                    # of Cre8or
```
Comment (sweetmantech) [inline components/LeaderboardPage/LeaderboardPage.tsx]:
```suggestion
                    Cre8or
```

File: lib/alchemy/eth_getLogs.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { error } from "console"
+/* eslint-disable no-console */
```
Comment (sweetmantech) [inline lib/alchemy/eth_getLogs.tsx]: can you disable only the line `console.error` is being used instead of muting for the entire file?

**Review by SweetmanTech — APPROVED** [review]
small comments 📝 

ship it 🚢

File: components/LeaderboardPage/LeaderboardPage.tsx (hunk 3 above)
Comment (towardsnewtech) [reply components/LeaderboardPage/LeaderboardPage.tsx:91]: fixed ✅

File: components/LeaderboardPage/LeaderboardPage.tsx (hunk 2 above)
Comment (towardsnewtech) [reply components/LeaderboardPage/LeaderboardPage.tsx:13]: updated ✅

**Review by SweetmanTech — APPROVED** [review]
LGTM 🙌

**Pull request merged**